embassy-stm32f4/Cargo.toml

```diff
@@ -39,3 +39,5 @@ cortex-m = "0.7.1"
 embedded-hal    = { version = "0.2.4" }
 embedded-dma    = { version = "0.1.2" }
 stm32f4xx-hal  = { version = "0.8.3", features = ["rt"], git = "https://github.com/stm32-rs/stm32f4xx-hal.git"}
+bxcan = {version = "0.4.0", git= "https://github.com/jonas-schievink/bxcan.git", branch = "defmt02" }
```

Comment:
Looks like this branch no longer exists but changes have been merged to
https://github.com/stm32-rs/bxcan

```toml
bxcan = { version = "0.4.0", git= "https://github.com/stm32-rs/bxcan.git", branch = "master" }
```